CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+## 2.0.0 - 2020-02-05
+
+### Added
+
+- Support for expire and get all Invoice
+- Support for Batch Disbursements
+- Support for create, get, update Fixed Virtual Account and get Fixed Virtual Account Payment
+- Support for create and get charge for Cards
+- Support for eWallets
+- Support for Retail Outlets
+- Support for Recurring Payments
+- Support for Payouts
+
+### Changed
+
+- Class structure - now the implementation is not merged into one class but divided into each product class
+- Request to HTTP Client now use Guzzle Client instead of cURL
```

Comment:
Does the user really need to know? What if they don't know about this?

Reply:
I think they need to know about HTTP client because the implication is in v2, it will add another dependency to guzzle library

Reply:
Alright cool